tutoraspects/commands_v0.py

```diff
@@ -107,6 +107,23 @@ def alembic(context, command) -> None:
     runner.run_job("aspects", command)
 
 
+# Ex: "tutor local do performance-metrics "
+@click.command(context_settings={"ignore_unknown_options": True})
+@click.pass_obj
+def performance_metrics(context) -> None:
+    """
+    Job to import Superset assets.
```

Comment:
Can you update this and describe what the command actually does? Same in the v1 version.